### ISSUE TITLE
Allow installing in a directory with `.git` and a readme file.

### ISSUE THREAD

**Issue by ranyitz** [wix/yoshi#492]
# 🙋 Feature Request

Enable the creation of new projects in an already cloned/initiated git repo.

### 😯 Current Behavior

Send a message that we `create-yoshi-app` only work on empty directories.

### 🤔 Expected Behavior

Be more graceful and enable in case these files are a `.git` directory and possible a `README.md` file, which GitHub sometimes suggest to create.

### 💁 Possible Solution

Allow creating projects in a directory that contains `.git` directory.

### 🔦 Context

The general way to generate a project is to use `create-yoshi-app <appName>` but some people prefer to first create the git repo, and only than, generate the project. We can allow this kind of use-case.

**Comment by netanelgilad** [comment]
My 2 cents on this problem. I also created a tool like yoshi in the past, and I added an `assert` command. The `assert` command could be used to generate everything into an existing directory, and it is also very useful for upgrades. If the generators were upgraded and now generate something new, just upgrading `yoshi` is not going to give you the new generated files. So an `assert` command could be used after upgrading `yoshi` to kind of regenerate the project into the current folder (not truly regenerate cause we don't want to override the changes to generated files, but more of an assertion that they exist and contain the minimum we need them to contain).

This would solve this issue, but it's probably a wider change and not a must for solving this issue.
